CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Features:
   `--scan-filenames/--no-scan-filenames` flag which allows users to enable or disable file name scanning.
 * [#254](https://github.com/godaddy/tartufo/pull/260) - Changes the default value of 
   `--regex/--no-regex` to True.
+* [#265](https://github.com/godaddy/tartufo/issues/265) - Adds new `--sensitivity`
```

Comment:
```suggestion
* [#265](https://github.com/godaddy/tartufo/issues/265) - Adds new `--entropy-sensitivity`
```

Reply:
Fixed. I missed this one after I forgot what was proposed and started off with the wrong option name. :-/